src/hazelcore/Builtins.re

```diff
@@ -0,0 +1,55 @@
+open EvaluatorResult;
+
+module Impl = {
+  [@deriving sexp]
+  type t = (list(DHExp.t), DHExp.t => EvaluatorResult.t) => EvaluatorResult.t;
+
+  let mk_one_arg = (f, ident, args, evaluate) => {
+    let e = DHExp.ApBuiltin(ident, args);
+    switch (args) {
+    | [] => Indet(e)
```

Comment:
should raise an exception if supplied an incorrect number of arguments, not just return Indet

Reply:
Fixed.